front_end/ndb/NdbMain.js

```diff
@@ -208,6 +238,26 @@ Ndb.NodeProcessManager = class extends Common.Object {
         info.id, userFriendlyName(info), SDK.Target.Type.Node,
         this._targetManager.targetById(info.ppid) || this._targetManager.mainTarget(), undefined, false, connection);
     target[NdbSdk.connectionSymbol] = connection;
+
+    try {
+      // this doesnt work
+      // target.runtimeAgent().invoke_evaluate({
+      //   expression: `
+      //     const zlib = require('http');
```

Comment:
@ak239 Is the script below will be evaluated in the inspected script's context? I followed your suggestion but it didn't work (I think it's because of the `import`)

Reply:
The easiest way to get proper `require` available is calling `Runtime.evaluate` with `includeCommandLineAPI ` flag:
```javascript
target.runtimeAgent().invoke_evaluate({
  expression: `require('zlib')`,
  includeCommandLineAPI: true
});
```

Reply:
I think this was what I missed. I will try it.

Reply:
It did work 🎉